src/animation/event.rs

```diff
@@ -47,6 +47,7 @@ fn update_animation_event_cache<T: FromReflect>(
     animation_clips: Res<Assets<AnimationClip2D>>,
 ) {
     for asset_event in asset_events.read() {
+        println!("{:?}", asset_event);
```

Comment:
Is this on purpose?

Reply:
Nah, I changed it to a warn! in the else path.
Still can not figure out why in your project it sometimes takes the else path to begin with though :D.

Reply:
Ah alright that's nice.

Though I didn't put a warn were you placed one, I had one here

```rust
        if max_event_frame > keyframes_len {
            return Err(AnimationClip2DError::InvalidFrame(
                max_event_frame,
                keyframes_len,
            ));
        } else if max_event_frame == keyframes_len {
            warn!("The last frame of an animation will not trigger any events!");
        }
```

you put a warn in the else, I just had else { continue; }.

Reply:
Yeah, I addressed that here https://github.com/KirmesBude/bevy_trickfilm/pull/33#discussion_r1817014654.
If you have 7 frames in the animation and specify an event at frame 7 it will now correctly throw this error.
Do you mind taking a look at that comment?

Reply:
Ah sorry, somehow missed that.